src/_posts/platform/getting-started/2000-01-01-getting-started-with-elk.md

```diff
@@ -385,6 +359,28 @@ The second environment variable is `LOGS_RETENTION_DAYS`. It configures
 the retention time of your logs (in days). Setting this variable to `10`,
 Curator will delete an index if it is 10+ days old.
 
+### Scheduling the task
```

Comment:
picky picky:
```suggestion
### Scheduling the Curator Task
```

Reply:
Oh yes, you are right, thanks!
Should be fixed with the second commit.